app/views/components/gds/exportsDateInput.scala.html

```diff
@@ -37,7 +37,7 @@
 }
 
 @dateInput(DateInput(
-  id = fieldName,
+  id = fieldName + "-input",
```

Comment:
In line 46 there is another ID which represents the whole fieldset. I think it's better to keep two ids (tests are based on those IDs) but to avoid duplication add "input" to input div.

The issue has been found in AXE report from accessibility tests